f2/src/components/formik/conditionalField/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import styled from '@emotion/styled'
+
+const FieldContainer = styled('div')`
+  border-left-width: 0.25rem;
+  border-left-color: rgb(174, 174, 174);
+  margin-left: 1.25rem;
+  padding-top: 1rem;
+  padding-left: 1.5rem;
+  width: -webkit-fill-available;
+`
+
+export const ConditionalField = (props) => {
```

Comment:
Do we need this layer? Sorry my brain kind of short circuit here.
I feel we only need FieldContainer. If you just export FieldContainer as ConditionalField, it can achieve same thing.

Reply:
Yes, you are right.